string camera_name
---
string dev_id